frontend/src/app/application-forms/tree-application-form/tree-permit-view/tree-permit-view.component.ts

```diff
@@ -0,0 +1,83 @@
+import { Component, OnInit, SecurityContext } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { Title } from '@angular/platform-browser';
+import { DomSanitizer } from '@angular/platform-browser';
+
+@Component({
+  selector: 'app-tree-permit-view',
+  templateUrl: './tree-permit-view.component.html'
+})
+export class TreePermitViewComponent implements OnInit {
+  forest: any;
+  permit: any = { totalCost: 0, quantity: 0, emailAddress: '' };
+  image: any;
+
+  constructor(
+    private route: ActivatedRoute,
+    private titleService: Title,
+    private sanitizer: DomSanitizer,
+    private router: Router
+  ) {}
+
+  ngOnInit() {
+    this.route.data.subscribe(data => {
+      if (data.permit && data.permit.forest) {
+        this.forest = data.permit.forest;
+        this.permit = data.permit;
+        this.titleService.setTitle(
+          'View your Christmas tree permit order confirmation for ' +
+            data.permit.forest.forestName +
+            ' National Forest | U.S. Forest Service Christmas Tree Permitting'
+        );
+        this.image = this.sanitizer.bypassSecurityTrustHtml(this.permit.permitImage);
+      } else {
+        this.router.navigate([`/`]);
+      }
+    });
+  }
+
+  printStyle() {
```

Comment:
does this have to be inline for printing? or can it be moved to another file?

Reply:
We moved that to a CSS file.